packages/dcos-integration-test/extra/test_remove_dockercfg_hook.py

```diff
@@ -0,0 +1,36 @@
+import uuid
+
+import pytest
+
+
+@pytest.fixture(scope='module')
+def auth_cluster(cluster):
```

Comment:
I think what we really want here is for this test to be run in at least one configuration. Cloud, on-prem, vagrant, whatever. Or all of them, it shouldn't matter where. What's the best way to configure pytest to do that? Maybe we rip out all these `conftest.py` (now `cluster_api.py`) changes and do something different.